### ISSUE TITLE
SQL Error "1062": Duplicate entry 

### ISSUE THREAD

**Issue by carlosgomesdefarianeto** [glpi-project/glpi#11400]
### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Version

10.0.0

### Bug description

Error message on update GLPi 9.5.7 to 10.0.0

![noname](https://user-images.githubusercontent.com/64043395/165384186-178adced-2846-445b-8fe2-3bb0c9ea33a2.png)

**Comment by cconard96** [comment]
It looks like you have a child entity with the same name as the root entity.

Please check the database for the bad data
```
SELECT `id`,`name` FROM glpi_entities WHERE id=0 OR entities_id=0;
```

The unicity check is to keep the complete name of all entities unique, but the root entity is a special case now. In the past, it may have had "-1" in its entities_id field which used to mean it had no parent. Since the migration to unsigned integers, this was updated to be 0 which points to itself. A side effect to this is that no entity that is a direct child of the root can have the same name as it.

**Comment by trasher** [comment]
No feedback from a while, closing.

**Comment by cedric-anne** [comment]
Fixed by #11776.
